### PR TITLE
added all required fields and methods

### PR DESCRIPTION
**Pull request by RLudchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,62 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String color;
+    private String model;
+    private int speed;
+    private boolean isRunning;
+    private core.basesyntax.Engine engine;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Car.java:8]:
```suggestion
    private Engine engine;
```

Comment (RLudchenko) [reply src/main/java/core/basesyntax/Car.java]: done

File: src/main/java/core/basesyntax/Car.java [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
 
-public class Car {
+public class Car implements Cloneable {
+    private String color;
+    private String model;
+    private int speed;
+    private boolean isRunning;
+    private Engine engine;
 
+    Car(String color, String model, int speed, boolean isRunning, core.basesyntax.Engine engine) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Car.java:10]:
```suggestion
    public Car(String color, String model, int speed, boolean isRunning, Engine engine) {
```

Comment (RLudchenko) [reply src/main/java/core/basesyntax/Car.java]: Sorry didn't notice, I don't know why it made it with core.basesyntax, but now I have changed everything!

**Pull request closed without merging**